types/react-table-config.d.ts

```diff
@@ -0,0 +1,90 @@
+import {
```

Comment:
Just curious where this came from/why it needs to be committed to git? Sorry for my ignorance

Reply:
Never be sorry! I added some explanation in a comment.
basically, react-table isn't written natively in Typescript so you need to get the types from DefinitelyTyped, which requires using this thing called declaration merging. I'm not entirely clear on why `yarn add @types/react-table` alone isn't enough, but that does seem to be outlined in https://stackoverflow.com/questions/62758670/something-went-wrong-with-react-table-using-typescript.